proof/refine/X64/Detype_R.thy

```diff
@@ -115,6 +118,7 @@ lemma deleteObjects_def2:
   "is_aligned ptr bits \<Longrightarrow>
    deleteObjects ptr bits = do
      stateAssert (deletionIsSafe ptr bits) [];
+     stateAssert (deletionIsSafe2 ptr bits) [];
```

Comment:
I don't get it, if we are asserting one immediately after the other, why do you need two?